start.sh

```diff
@@ -0,0 +1 @@
+git pull upstream master && git push -f && git checkout -b husni/patch-003
```

Comment:
Same here.

Reply:
sorry for that, I have carefully tried not to commit them :(